packages/subgraph/manifest.template.yaml

```diff
@@ -0,0 +1,26 @@
+specVersion: 0.0.2
+description: Balancer Pool Metadata Registry provides a registry for Balancer's Pools metadata.
+repository: https://github.com/bleu-llc/balancer-pool-metadata
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: PoolMetadataRegistry
+    network: {{network}}
+    source:
+      address: "{{PoolMetadataRegistry.address}}"
+      abi: PoolMetadataRegistry
+      startBlock: {{PoolMetadataRegistry.startBlock}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      entities:
+        - PoolMetadataUpdated
+      abis:
+        - name: PoolMetadataRegistry
+          file: ./abis/PoolMetadataRegistry.json
+      eventHandlers:
+        - event: PoolMetadataUpdated(indexed bytes32,string)
+          handler: handlePoolMetadataUpdated
+      file: ./src/mappings/pool-metadata-registry.ts
```

Comment:
just want to make sure kebab case is the conventional here @mendesfabio ?

Reply:
not sure if there's a convention but I'm used to camel case.

Reply:
I changed to camelCase ✌🏻